### PR TITLE
add new base jupyter/R image

### PR DESCRIPTION
**Pull request by wangfan860**
Jira Ticket: [PXP-8340](https://ctds-planx.atlassian.net/browse/PXP-8340)
### New Features
Add new Jupyter/R image with latest python and R version.

### CODE REVIEW

File: jupyter-6.4.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+FROM quay.io/cdis/scipy-notebook:f1a4a37136d8
```
Comment (paulineribeyre) [inline jupyter-6.4.0/Dockerfile:3]: why not `notebook-6.4.0` instead of `f1a4a37136d8`, for clarity

File: jupyter-6.4.0/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# TL;DR
+
+Jupyter/R base image with Python 3.9.5 and R 4.1.0 and docker-compose suitable for deployment to Gen3 hatchery. Please note this iamge is based upon `quay.io/cdis/scipy-notebook:f1a4a37136d8` (originally pulled from Docker Hub).
```
Comment (paulineribeyre) [inline jupyter-6.4.0/README.md:3]: same comment about the version

**Review by paulineribeyre — APPROVED** [review]

File: jupyter-6.4.0/Dockerfile (hunk 1 above)
Comment (wangfan860) [reply jupyter-6.4.0/Dockerfile:3]: sure, I will update😊

File: jupyter-6.4.0/README.md (hunk 2 above)
Comment (wangfan860) [reply jupyter-6.4.0/README.md:3]: i will update😊

**Pull request merged**